test/e2e/basic/default_irq_smp_affinity.go

```diff
@@ -0,0 +1,84 @@
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	coreapi "k8s.io/api/core/v1"
+
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+	util "github.com/openshift/cluster-node-tuning-operator/test/e2e/util"
+)
+
+// Test the application (and rollback) of [scheduler]'s default_irq_smp_affinity option.
+// See: https://github.com/redhat-performance/tuned/pull/306
+var _ = ginkgo.Describe("[basic][default_irq_smp_affinity] Node Tuning Operator set irq default smp affinity", func() {
+	const (
+		profileAffinity           = "../testing_manifests/default_irq_smp_affinity.yaml"
+		nodeLabelAffinity         = "tuned.openshift.io/default-irq-smp-affinity"
```

Comment:
This label is only used bu the test, right?

Reply:
Yes. it needs to match `default_irq_smp_affinity.yaml`.  I should probably add a comment.

Reply:
thanks